server/src/models/Review.js

```diff
@@ -4,7 +4,6 @@ class Review extends Model {
     static get tableName() {
         return "reviews"
     }
-
     static get jsonSchema() {
```

Comment:
there should be a new empty line between blocks of each method
https://github.com/airbnb/javascript?tab=readme-ov-file#whitespace--after-blocks

Reply:
done